build.assets/tooling/cmd/goderive/plugin/deepcopy/0001-add-time-Time-deepcopy-support.patch

```diff
@@ -0,0 +1,21 @@
+--- a/deepcopy.go
++++ b/deepcopy.go
+@@ -142,6 +142,17 @@ func (g *gen) genStatement(typ types.Type, this, that string) error {
```

Comment:
@tigrato Could you share the full patch of the `deepcopy.go` ? 

I see that some  other logic in `deepcopy.go` was reworked in you file shared in slack.

Reply:
The patch is correct but you picked the master version of the file while we use the v0.5.1 tag.

Adapt the script to 

```
GODERIVE_VERSION=$(go list -m -f '{{.Version}}' github.com/awalterschulze/goderive)
curl -fsSL \
 "https://raw.githubusercontent.com/awalterschulze/goderive/refs/tags/${GODERIVE_VERSION}/plugin/deepcopy/deepcopy.go" \
 -o deepcopy.go
```